packages/runtime/container-runtime/src/containerHandleContext.ts

```diff
@@ -42,4 +45,8 @@ export class ContainerFluidHandleContext implements IFluidHandleContext {
     public async resolveHandle(request: IRequest): Promise<IResponse> {
         return this.runtime.resolveHandle(request);
     }
+
+    public handleDecoded(url: string): void {
```

Comment:
It feels to me that it's better to track handles themselves than URIs.
Or if we do URIs, we will need to clearly prescribe what URI means.
For example, if https://github.com/microsoft/FluidFramework/issues/6001 were to happen, will it be full URI, or first part of it? What if handles evolve over time and are structured somehow differently (not sure how, but let's say they have 3 parts :)). I feel layers like that should not care, and only GC should reason about what info it needs to extract from handle.

Reply:
That makes sense. I can change this to handles.